mdpow/analysis/workflows/base.py

```diff
@@ -0,0 +1,156 @@
+# MDPOW: base.py
+# 2022 Cade Duckworth
+
+"""
+.. module:: mdpow.analysis.workflows.base --- Base functions for use with all workflow modules
+===============================================================================================
+"""
+
+import os
+import re
+import pandas as pd
+
+#import will need to change based on new naming convention for ada
+#and once the function changes for more general use
+import mdpow.analysis.workflows.dihedrals as ada
```

Comment:
```python
import .dihedrals as ada
```
when importing from the same package

Reply:
However, base packages typically don't import other but instead are imported, this avoids circular imports.

Perhaps "base" is not the right name then.

Reply:
Perhaps automation or project automation? Workflows/dihedrals is the automated version of dihedral analysis for one project but workflows/base iterates that automation through a set of projects so it is the top-level automation.